ex01.ex

```diff
@@ -86,7 +86,7 @@ defmodule Ex01 do
   # function. The examples below will make this clearer :)               #
   ########################################################################
 
-  create_adder = your_anonymous_function(1)
+  create_adder = fn incr -> &(&1 + incr) end
```

Comment:
This is just a philosophical observation. Sometimes you put parens around the formal parameter list, and sometimes you don't.

I find myself doing the same thing. And I've never found a pattern.